src/shared/pxToRem.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Convert px to rem
+ */
+export default function pxToRem(px: number | string, base = 16): string {
```

Comment:
What is the difference between the library you have imported and this function? Do we need both?

Reply:
I'm sorry what? There is a lib?